nuxeo-csv-core/src/main/java/org/nuxeo/ecm/csv/core/CSVImportStatus.java

```diff
@@ -27,49 +27,88 @@
  */
 public class CSVImportStatus implements Serializable {
 
+    public enum State {
+        SCHEDULED, RUNNING, COMPLETED,
+        /**
+         * @since 9.1
+         */
+        ERROR
+    }
+
     private static final long serialVersionUID = 1L;
 
     private final State state;
 
-    private final int positionInQueue;
+    @Deprecated
```

Comment:
Yes any `@Deprecated` annotation must be paired with a `@deprecated since ...` Javadoc.